### PR TITLE
added cluster node timeout config

### PR DESCRIPTION
**Pull request by DvirDukhan**

### CODE REVIEW

**Review by MeirShpilraien — COMMENTED** [review]
@DvirDukhan you probably want to increase this value depends on how you run (valgrind) and not from the test itself, I do not mind leaving it also to the test but we also (probably) want to give it as an argument to RLTest to be applied globally on all tests.

**Comment by DvirDukhan** [comment]
This was the intention of the test. What did I miss? @MeirShpilraien

**Comment by MeirShpilraien** [comment]
@DvirDukhan I mean that we should also allow set it when you run RLTest, `RLTest --cluster-node-timeout 60000`

**Comment by rafie** [comment]
@MeirShpilraien @DvirDukhan Please don't forget to update the RLTest version.

File: RLTest/__main__.py [hunk 1]
```diff
@@ -133,6 +133,10 @@ def do_normal_conn(self, line):
     '--oss_password', default=None,
     help='set redis password, relevant for oss and oss-cluster environment')
 
+parser.add_argument(
+    '--cluster_node_timeout', default=5000,
+    help='sets the node timeout on cluster')
```
Comment (rafie) [inline RLTest/__main__.py:138]: units? ms?

**Review by rafie — APPROVED** [review]

**Pull request merged**